app/src/main/java/campus/tech/kakao/map/MainActivity.kt

```diff
@@ -1,11 +1,136 @@
 package campus.tech.kakao.map
 
+import android.content.Context
+import android.content.Intent
 import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
+import android.util.Log
+import android.widget.EditText
+import android.widget.ImageView
+import android.widget.TextView
 import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.isGone
+import androidx.core.view.isVisible
+import androidx.databinding.DataBindingUtil
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import campus.tech.kakao.map.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityMainBinding
+    private lateinit var mainModel: MainModel
+    private lateinit var mainViewModel: MainViewModel
+
+    private lateinit var input: EditText
+    private lateinit var researchCloseButton: ImageView
+    private lateinit var noResultTextView: TextView
+    private lateinit var recyclerView: RecyclerView
+    private lateinit var tabRecyclerView: RecyclerView
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        Log.d("mytest", "MainAcitivty_onCreate")
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_main)
+
+        mainModel = MainModel(application as MyApplication)
+
+        val viewModelFactory = MainViewModelFactory(application as MyApplication, mainModel)
+        mainViewModel = ViewModelProvider(this, viewModelFactory)[MainViewModel::class.java]
+
+        binding.viewModel = mainViewModel
+        binding.lifecycleOwner = this
+
+        input = binding.input
+        researchCloseButton = binding.closeButton
+        noResultTextView = binding.noResultTextview
+        recyclerView = binding.recyclerView
+        tabRecyclerView = binding.tabRecyclerview
+
+        val resultAdapter = RecyclerViewAdapter {
+            mainViewModel.resultItemClickListener(it)
+            moveMapView(it)
+        }
+        recyclerView.adapter = resultAdapter
+        recyclerView.layoutManager = LinearLayoutManager(this)
+
+        mainViewModel.placeList.observe(this) { list ->
+            if (list.isEmpty()) {
+                noResultTextView.isVisible = true
+                recyclerView.isGone = true
```

Comment:
```suggestion
                recyclerView.isVisible = false
```
로 하나만 사용하시는게 가독성엔 더 나아 보입니다! (필수 변경사항은 아닙니다)